src/services/has-many-getter.js

```diff
@@ -80,26 +80,38 @@ function HasManyGetter(model, association, opts, params) {
         var query = association.find(conditions);
         handlePopulate(query);
 
-        return query;
+        return query.then(function(records) {
```

Comment:
Can you add a description in the changelog for the fix you made in this PR?

Reply:
done